app/main.py

```diff
@@ -1,3 +1,65 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, kilometers: int) -> None:
+        self.km = kilometers
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(
+            self, new_distance: Union["Distance", int, float]
```

Comment:
PLease correct annotation to Distance | int | float
correct in places below as well
use `from __future__ import annotation` if needed